lib/timeline/Core.js

```diff
@@ -677,6 +677,46 @@ Core.prototype.getWindow = function() {
 };
 
 /**
+ * Zoom in the window such that given time is centered on screen.
+ * @param {Number} percentage - must be between [0..1]
+ */
+Core.prototype.zoomIn = function(percentage) {
+  if (!percentage || percentage < 0 || percentage > 1) return
```

Comment:
I don't like the (non existing) error handling. But as far as I it is not better in all the other functions so that's ok for now.